recipes/vyper/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "vyper" %}
+{% set version = "0.2.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vyper-{{ version }}.tar.gz
+  sha256: b926ea7e3100064bdd91540ee1b444a4d5d1a43cb080b35dc05eb3c8f95b3ed6
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - vyper=vyper.cli.vyper_compile:_parse_cli_args
+    - vyper-serve=vyper.cli.vyper_serve:_parse_cli_args
+    - vyper-lll=vyper.cli.vyper_lll:_parse_cli_args
+    - vyper-json=vyper.cli.vyper_json:_parse_cli_args
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+  run:
+    - asttokens ==2.0.4
+    - pycryptodome >=3.5.1,<4
+    - python >=3.6
+    - semantic_version ==2.8.5
+
+test:
+  imports:
+    - vyper
+    - vyper.ast
+  commands:
+    - pip check
+    - vyper --help
+    - vyper-serve --help
+    - vyper-lll --help
+    - vyper-json --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/vyperlang/vyper
+  summary: 'Vyper: the Pythonic Programming Language for the EVM'
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - spinoch
```

Comment:
@vyperlang Do any of you also want to be added here?

Reply:
sure add @fubuloubu and @iamdefinitelyahuman

Reply:
@iamdefinitelyahuman It looks like I need your explicit acknowledgement to add you here

Reply:
Explicitly acknowledged and approved.